pytorch_toolkit/light-weight-face-anti-spoofing/datasets/casia_surf.py

```diff
@@ -0,0 +1,87 @@
+"""
+ Copyright (c) 2020 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import os
+import re
+
+import cv2 as cv
+import numpy as np
+import torch
+from torch.utils.data import Dataset
+
+
+class CasiaSurfDataset(Dataset):
+    def __init__(self, protocol: int, dir_: str = 'data/CASIA_SURF', mode: str = 'train', depth=False, ir=False,
+                 transform=None):
+        self.dir = dir_
+        self.mode = mode
+        submode = {'train': 'train', 'dev': 'dev_ref',
```

Comment:
Move `dict` definition to the class level: `class CasiaSurfDataset(Dataset): submodes = {...} def __init__() ...`